common/common-client/src/main/java/org/ow2/proactive/authentication/FileLoginModule.java

```diff
@@ -258,21 +256,17 @@ private boolean authenticateUserFromFile(String username, String password) throw
      * @throws LoginException if group file is not found or unreadable.
      */
     protected void groupMembershipFromFile(String username) throws LoginException {
-        Properties groups = new Properties();
-        try {
+
+        try (FileInputStream stream = new FileInputStream(groupFile)) {
+            BufferedReader reader = new BufferedReader(new InputStreamReader(stream, Charsets.UTF_8));
```

Comment:
It could be interesting to mention in the documentation that UTF-8 must be used for configuration files.